ooiui/static/js/views/asset_management/AssetView.js

```diff
@@ -6,13 +6,20 @@ var ParentAssetView = Backbone.View.extend({
      *  - derender()
      */
     initialize: function() {
-        _.bindAll(this, 'render', 'derender');
+        "use strict";
```

Comment:
@birdage , at least that's what Lint says lol
![screen shot 2015-11-04 at 3 10 56 pm](https://cloud.githubusercontent.com/assets/3743882/10950580/51666f22-8306-11e5-9b34-9c56d5d29153.png)